### PR TITLE
Add POPITEM opcode

### LINKED ISSUE #1664: Add POPITEM opcode
I'm not sure we have any use for it in the compiler, but it of course needs to be supported in the VM, see neo-project/neo-vm#387.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1664 .

### CODE REVIEW

File: pkg/vm/vm.go [hunk 1]
```diff
@@ -1216,6 +1216,19 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
...
+		arr := v.estack.Pop().Item()
+		elems := arr.Value().([]stackitem.Item)
+		index := len(elems) - 1
+		elem := elems[index]
+		switch item := arr.(type) {
+		case *stackitem.Array:
+			item.Remove(index)
+		case *stackitem.Struct:
+			item.Remove(index)
+		}
```
Comment (roman-khimov) [inline pkg/vm/vm.go:1229]: `v.refs.Remove()`?

File: pkg/compiler/codegen.go [hunk 2]
```diff
@@ -852,7 +852,7 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 				// This way even non-pointer struct fields will be copied.
 				emit.Opcodes(c.prog.BinWriter, opcode.NEWARRAY0,
 					opcode.DUP, opcode.ROT, opcode.APPEND,
-					opcode.PUSH0, opcode.PICKITEM)
+					opcode.POPITEM)
```
Comment (roman-khimov) [inline pkg/compiler/codegen.go:855]: Wouldn't it reverse item order?

Comment (fyrchik) [reply pkg/compiler/codegen.go:855]: What do you mean? We have array of 1 element -- the struct we need to copy.

File: pkg/vm/vm.go (hunk 1 above)
Comment (fyrchik) [reply pkg/vm/vm.go:1229]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**